server/src/tasks/relay.js

```diff
@@ -118,15 +118,11 @@ const relay = withWalletAccount(async (account, { walletAddress, methodName, met
       }
     })
 
-    const returnValues = receipt && receipt.events.TransactionExecuted.returnValues
-    if (!returnValues) {
-      job.attrs.data.transactionBody = { ...lodash.get(job.attrs.data, 'transactionBody', {}), status: 'failed' }
-      job.save()
-      throw new Error(`No return values in receipt (or now receipt)`)
-    }
-    const { success, wallet, signedHash } = returnValues
-    const { blockNumber } = receipt
+    const success = lodash.get(receipt, 'events.TransactionExecuted.returnValues.success')
```

Comment:
@LiorAgnin  didn't know this is exists too,
I referred to checking receipt.status- https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#eth-gettransactionreceipt-return 😅
This check is cool but please add this too